internal/servers/repository_worker_test.go

```diff
@@ -551,3 +551,189 @@ func TestRepository_CreateWorker(t *testing.T) {
 		})
 	}
 }
+
+func TestRepository_UpdateWorker(t *testing.T) {
```

Comment:
in general: it might be nice to test unhappy paths since I think it's fairly trivial to test all of them, even without resorting to sqlmock (see CreateWorker tests).